Web/registration.js

```diff
@@ -0,0 +1,98 @@
+// JavaScript Document
+function validate(){
+	
+	var pwd= document.getElementById('password').value; 
+	var re_pwd= document.getElementById('re_password').value; 
+	if(pwd!=re_pwd)
+		{
+			alert("Not same password and confirm password");
+			return false;
+			
+		}
+	var age_value=document.getElementById('age').value;
+	var filter = /^[0-9]+(.[0-9]{1,2})?$/;
+	if (!filter.test(age_value)) 
+		{
+			alert('Please provide a valid age');
+			return false;
+		}
+	else{
+			alert('User Created Successfully');
+			return true;
+		}
+ 
+ 
+		
+}
+
+function byId(e) {
+	return document.getElementById(e);
+	}
+ 
+function stateComboChange()
+    {
+        var combo1 = byId('stateCombo');
```

Comment:
ACTION: It is fine, but for learning purpose keep your data in JSON and then add options accordingly.